integration-test/src/test/java/org/apache/iotdb/confignode/it/cluster/IoTDBClusterRestartIT.java

```diff
@@ -50,15 +51,14 @@
 @RunWith(IoTDBTestRunner.class)
 @Category({ClusterIT.class})
 public class IoTDBClusterRestartIT {
+  private static final Logger logger = IoTDBTestLogger.logger;
 
   private static final String ratisConsensusProtocolClass =
       "org.apache.iotdb.consensus.ratis.RatisConsensus";
-  private static final int testConfigNodeNum = 2;
-  private static final int testDataNodeNum = 2;
+
   private static final int testReplicationFactor = 2;
 
-  @Before
-  public void setUp() throws Exception {
+  public void setUp(int configNodeNum, int dataNodeNum) {
```

Comment:
How about just use 3c2d?

Reply:
Agree, it's ok that all 3 tests use 3c2d.